From 0de788aa6175fa6035b9f79a7dcfda8b98cd1e6f Mon Sep 17 00:00:00 2001
From: Frediano Ziglio <fziglio@redhat.com>
Date: Fri, 25 May 2018 19:41:26 +0100
Subject: [PATCH 13/43] file_xfer: Remove FileXferTask structure alignment

There's no reason beside losing performances to align
that structure, is not passed as binary data.

Signed-off-by: Frediano Ziglio <fziglio@redhat.com>
Acked-by: Christophe Fergeau <cfergeau@redhat.com>
---
 vdagent/file_xfer.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/vdagent/file_xfer.h b/vdagent/file_xfer.h
index 25cd5c2..029d0e7 100644
--- a/vdagent/file_xfer.h
+++ b/vdagent/file_xfer.h
@@ -21,7 +21,7 @@
 #include <map>
 #include "vdcommon.h"
 
-typedef struct ALIGN_VC FileXferTask {
+typedef struct FileXferTask {
     FileXferTask(HANDLE _handle, uint64_t _size, const TCHAR* _name):
     handle(_handle), size(_size), pos(0) {
         // FIXME: should raise an error if name is too long..
@@ -36,7 +36,7 @@ typedef struct ALIGN_VC FileXferTask {
     TCHAR name[MAX_PATH];
 
     void cancel();
-} ALIGN_GCC FileXferTask;
+} FileXferTask;
 
 typedef std::map<uint32_t, FileXferTask*> FileXferTasks;
 
-- 
2.17.1

